.env.template.testnet.gobi

```
@@ -3,8 +3,25 @@ COMPOSE_PROJECT_NAME=evmapp-testnet-gobi
 INTERNAL_NETWORK_SUBNET='10.10.50.0/24'
 SCNODE_IP_ADDRESS='10.10.50.3'
 
+ZEN_TAG=v4.1.1
+ZEN_PORT=19033
+ZEN_RPC_USER=zenuser
+ZEN_RPC_PASSWORD=password
+ZEN_RPC_PORT=8231
```

```suggestion
ZEN_RPC_PORT=18231
```
Let's use network specific default ports.